### DISCUSSION TITLE
Some new features

### DISCUSSION THREAD

**Discussion by invincible791** [Nain57/Smart-AutoClicker#255] [General Feature Requests]
I want to know if it's possible to implement a scenario that only appears after a certain number of loops, such as executing a certain scene after ten cycles. Thank you.

**Comment by Nain57** [comment]
Closed as implemented (a long ago)
Multilple solution of flow control are now available, check the [wiki](https://github.com/Nain57/Smart-AutoClicker/wiki)!
